phoenix-core/src/main/java/org/apache/phoenix/query/ConnectionQueryServicesImpl.java

```diff
@@ -1546,17 +1538,14 @@ public GetVersionResponse call(MetaDataService instance)
                 MetaDataUtil.ClientServerCompatibility compatibility = MetaDataUtil.areClientAndServerCompatible(serverJarVersion);
                 if (!compatibility.getIsCompatible()) {
                     if (compatibility.getErrorCode() == SQLExceptionCode.OUTDATED_JARS.getErrorCode()) {
-                        HRegionLocation name = regionMap.get(result.getKey());
                         errorMessage.append("Newer Phoenix clients can't communicate with older "
                                 + "Phoenix servers. Client version: "
                                 + MetaDataProtocol.CURRENT_CLIENT_VERSION
                                 + "; Server version: "
                                 + getServerVersion(serverJarVersion)
                                 + " The following servers require an updated "
                                 + QueryConstants.DEFAULT_COPROCESS_JAR_NAME
-                                + " to be put in the classpath of HBase: ");
-                        errorMessage.append(name);
-                        errorMessage.append(';');
+                                + " to be put in the classpath of HBase; ");
```

Comment:
Change this error message log now that we aren't mentioning the region name. Instead of "The following servers .." we can say "Servers require an updated .. "

Reply:
Changed the error message.